### PR TITLE
Changes for Sandbox Onboarding

### PR DESCRIPTION
**Pull request by techmaharaj**
- [X] Create a maintainer list
- [X] GitHub: ensure that hat the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub
- [X] CII: Start on a CII best practices badge https://bestpractices.coreinfrastructure.org/en
- [x] GitHub: ensure [DCO](https://github.com/apps/dco) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Paralus Maintainers
+
+[GOVERNANCE.md](/paralus/GOVERNANCE.md) describes the Paralus governance.
+
+## Maintainers
```
Comment (akshay196) [inline MAINTAINERS.md:5]: @techmaharaj  Few comments about Maintainers:
1. Generally I have seen such kind of list is maintained in an alphabetical order of person's names. Did you following some different ordering here? (I don't wish to move my name above in the list :P but I definitely wanted to keep list consistent and structured as we keep adding more maintainers in the future)
2. Have you referred any list from other projects for Project Roles? Generally who is reviewer also has permission to approve the request.

Comment (techmaharaj) [reply MAINTAINERS.md:5]: There isn't any specific order specified, I did refer to [Argo's list](https://github.com/argoproj/argoproj/blob/master/MAINTAINERS.md) just kept Rafay folks on the top followed by us.

Comment (akshay196) [reply MAINTAINERS.md:5]: @techmaharaj  Regarding Project Roles, have you referred any document for roles? Should we define what exactly is Reviewer, Approver and other roles we have for Paralus project?

Comment (techmaharaj) [reply MAINTAINERS.md:5]: I feel the reviewer and approver roles are understood. Do we have any other roles at the moment? If yes, let me know and I'll try to update this. 

My thought is, we can do such changes in the next iteration/update & let this move to cncf maintainers list as is at the moment and we could refine it later.

**Review by niravparikh05 — DISMISSED** [review]
lgtm

**Review by niravparikh05 — APPROVED** [review]
lgtm

**Pull request merged**